doc/env_vars.md

```diff
@@ -117,5 +117,6 @@ You may have noted that some options are not available:
 | `LRSQL_OIDC_CLIENT_ID` | `oidcClientId` | An optional OIDC client ID for the SQL LRS Admin SPA. If provided, along with the `LRSQL_OIDC_ISSUER` and `LRSQL_OIDC_AUDIENCE` variables, will enable OIDC access to the Admin UI. | Not Set |
 | `LRSQL_OIDC_CLIENT_TEMPLATE` | `oidcClientTemplate` | An optional template to modify LRS Admin UI client OIDC configuration. | Not Set |
 | `LRSQL_OIDC_VERIFY_REMOTE_ISSUER` | `oidcVerifyRemoteIssuer` | Verify on startup that the issuer in remote configuration matches `LRSQL_OIDC_ISSUER`. No effect if `LRSQL_OIDC_ISSUER` is not set. | `true` |
+| `LRSQL_OIDC_ENABLE_LOCAL_ADMIN` | `oidcEnableLocalAdmin` | Whether or not to enable local administrative account login when `LRSQL_OIDC_CLIENT_ID` is set. | `false` |
```

Comment:
Since this var only takes effect if `LRSQL_OIDC_CLIENT_ID` is set, that means this wouldn't have an effect if the user is logging in via OIDC _without_ using the admin UI (e.g. they're CURL-ing into the LRS). How important would taking into account this edge case be?

Reply:
Super important! I actually brought this up on Slack this morning before reading this, you're spot on. There are a couple things that need coverage/thought here:
* This doc is actually incorrect, because setting this to false will actually prevent any local admin login, even if an `oidc-client-id` is not present. I'll update that.
  * This makes the admin FE effectively useless if `oidc-client-id` is not also provided, but a non-functional login screen is fine
* The local admin account CRUD still works for direct OIDC token users when `oidc-enable-local-admin` is false, this is not desirable.

Reply:
doc update https://github.com/yetanalytics/lrsql/pull/217/commits/dbf89319e5a84e7b6ee053fe532fe5a5c6c00d29

Reply:
disable account routes: https://github.com/yetanalytics/lrsql/pull/217/commits/fa1ffbbaa996d6f9191aa068881f487793ac7e68